prime-router/src/testIntegration/resources/datatests/mappinginventory/cx/cx-identifier-organization.hl7

```diff
@@ -1,3 +1,2 @@
-MSH|^~\&#|||||20230501102531-0400||ORU^R01^ORU_R01|12345||2.5.1|||||USA|UNICODE UTF-8
-OBR|1|two|three|68991-9^Epidemiologically Important Information^LN^^^^^^^^^^^2.16.840.1.113883.6.1|||||||||||||||||||||unknown||||||||||||||||||||||||||||IDNumber^checkdigit^checkdigitscheme^blah&blah&CLIA^PLAC^blah2&blah2&CLIP^19880704^20880704^Assigning Jurisdiction^Assigning Agency^SecurityCheck^VID
-OBX|1|DT|11368-8^Illness or injury onset date and time^LN^^^^2.61||20230201||||||F|||20230227|11D0668319||||20230227||||CDC Atlanta Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^11D0668319|1600 Cliffton Road^^Atlanta^GA^30329^USA^B
+MSH|^~\&#|||||20230501102531-0400||ORU^R01^ORU_R01|12345||2.5.1
+OBR|1|two|three|68991-9^Epidemiologically Important Information^LN^^^^^^^^^^^2.16.840.1.113883.6.1|||||||||||||||||||||unknown||||||||||||||||||||||||||||IDNumber^checkdigit^checkdigitscheme^blah&blah&CLIA^PLAC^blah2&blah2&CLIP^19880704000000+0000^20880704000000+0000^AJCode&Assigning Jurisdiction&AJSys^AACode&Assigning Agency&AASys
```

Comment:
We should get rid of the `000000+0000`s in here to be able to verify that the new dateTime mappings are working correctly.